nemo_skills/inference/model/vllm_multimodal.py

```
@@ -108,3 +148,298 @@ def _process_audio_response(self, audio_data, response_id: str) -> dict:
+        # Extract from single quotes (greedy - matches LONGEST)
+        match = re.search(r"'(.*)'", generation)
+        if match:
+            generation = match.group(1)
+
+        return generation
+
+    async def generate_async(
+        self,
+        prompt: str | list[dict] | None = None,
+        tokens_to_generate: int | None = None,
+        task_type: str = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)